akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolConductor.scala

```diff
@@ -38,18 +38,22 @@ private object PoolConductor {
       Ports(
         inlets.head.asInstanceOf[Inlet[RequestContext]],
         inlets.last.asInstanceOf[Inlet[RawSlotEvent]],
-        outlets.asInstanceOf[immutable.Seq[Outlet[RequestContext]]])
+        outlets.asInstanceOf[immutable.Seq[Outlet[SlotCommand]]])
+  }
+
+  case class PoolSlotsSetting(minSlots: Int, maxSlots: Int) {
+    require(minSlots <= maxSlots, "min-connections must be <= max-connections")
   }
 
   /*
     Stream Setup
     ============
-                                                                                                  Request-
-    Request-   +-----------+     +-----------+    Switch-    +-------------+     +-----------+    Context
+                                                                                                  Slot-
+    Request-   +-----------+     +-----------+    Switch-    +-------------+     +-----------+    Command
     Context    |   retry   |     |   slot-   |    Command    |   doubler   |     |   route   +-------------->
     +--------->|   Merge   +---->| Selector  +-------------->| (MapConcat) +---->|  (Flexi   +-------------->
                |           |     |           |               |             |     |   Route)  +-------------->
-               +----+------+     +-----+-----+               +-------------+     +-----------+       to slots
+               +----+------+     +-----+-----+               +-------------+     +-----------+       to slotSettings
```

Comment:
I guess IDEA might have changed it in when I was doing auto-rename. Will bring the old name back